client/src/components/Header/Header.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function Header() {
```

Comment:
I don't see an index.js file with this component. Have you tested this locally to see if it works?

Reply:
I will double check.

Reply:
Take a look at the other components. Use those as an example.